regress/github-#0260/run

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+# This regression test is a part of SIPp.
+# Author: Pietro Bertera, Snom Technology AG, 2016
+
+. "`dirname "$0"`/../functions"; init
+
+sippbg -m 1 -sf uas.xml -p 5070 -i 127.0.0.1 \
+    -timeout 4 -timeout_error -trace_logs \
+    -log_file log.log >/dev/null 2>&1
+
+job=$!
+
+sippfg -m 1 -sf uac.xml 127.0.0.1:5070 \
+    -timeout 4 -timeout_error >/dev/null 2>&1
+
+status=$?
+wait $job || status=1
+
+if test $status -eq 0; then
+    if grep -e ^LAST\ From:\ \"Tom\ Jones\"\ \<sip:tom.jones@wales.uk\>\;tag=SIPpTag001$ log.log > /dev/null; then
+        if grep -e ^LAST\ To:uri:\ sip:cheese@paris.fr$ log.log > /dev/null; then
+            if grep -e ^LAST\ From-tag:\ SIPpTag001$ log.log > /dev/null; then
+                ok
+            else
+                fail "From:param.tag not found"
+            fi
+        else
+            fail "To:uri not found"
+        fi
+    else
+        fail "From header not found"
+    fi
+else
+    fail "process failure"
+fi
```

Comment:
This could be done without so much nesting with negative tests instead:
```
if test $status -ne 0; then
    fail "process failure"
elif ! grep -q '^LAST From: "Tom Jones" <sip:tom\.jones@wales.uk>;tag=SIPpTag001$' log.log; then
    fail "From header not found"
...
```